include/crow/middleware_context.h

```diff
@@ -24,16 +26,22 @@ namespace crow
             }
         };
 
+
+
         template <>
         struct partial_context<>
         {
             template <int>
             using partial = partial_context;
         };
 
+
+
```

Comment:
Missing comment or just additional blank lines?

Reply:
additional blank lines (first time I opened the file I thought there was only 1 class)